components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -354,6 +354,11 @@ interface AIChatUIHandler {
   // so that we can use [EnableIfNot=is_android] here.
   OpenConversationFullPage(string conversation_uuid);
 
+  // Determines whether a conversation exists - we need this so we can determine
+  // whether a URL is bogus (i.e. made up) or if it corresponds to a
+  // not-visible-in-sidebar conversation.
+  ConversationExists(string conversation_uuid) => (bool exists);
```

Comment:
I'm not sure why this needs to be proxied by the AIChatUIHandler instead of just on the Service directly?

Reply:
Good call! I guess I was just on autopilot!